content/departments/support/process/permanently_deleting_user_data.md

```diff
@@ -18,13 +18,15 @@ This will involve coordinating with multiple teams to get all the data deleted f
 
 ### In the product: how to nuke a user account
 
-If the request is for a Sourcegraph.com account, engage [one of your teammates who has access](../tools/index.md#sourcegraphcom-admin-access) (it requires site admin).
+1a. If the request is for a Sourcegraph.com account, engage [one of your teammates who has access](../tools/index.md#sourcegraphcom-admin-access) (it requires site admin). Inform the customer that you can only remove the username associated with the email they submitted the request from.
```

Comment:
This looks like a fine addition to me, you might consider adding a caveat for the hypothetical scenario where a person requests an account to be deactivated from an email that does not match the user email, in that scenario we should not reveal the user email to the "nuke" requestor. As this would be a way for malicious actors to gain information about Sourcegraph.com users.